test/e2e/testnet/node.go

```diff
@@ -292,52 +303,9 @@ func (n *Node) Upgrade(version string) error {
 		return err
 	}
 
-	return n.forwardPorts()
-}
-
-func (n *Node) forwardPorts() error {
-	rpcProxyPort, err := n.Instance.PortForwardTCP(rpcPort)
-	if err != nil {
-		return fmt.Errorf("forwarding port %d: %w", rpcPort, err)
-	}
-
-	grpcProxyPort, err := n.Instance.PortForwardTCP(grpcPort)
-	if err != nil {
-		return fmt.Errorf("forwarding port %d: %w", grpcPort, err)
-	}
-
-	n.rpcProxyPort = rpcProxyPort
-	n.grpcProxyPort = grpcProxyPort
-
-	return nil
-}
-
-func (n *Node) ForwardBitTwisterPort() error {
```

Comment:
This is not needed. When using `instance.EnableBittwister` this is done under the hood.